db/filename_test.cc

```diff
@@ -171,6 +171,68 @@ TEST_F(FileNameTest, Construction) {
   ASSERT_EQ(kMetaDatabase, type);
 }
 
+TEST_F(FileNameTest, NormalizePath) {
+
+  std::string separator(1, kFilePathSeparator);
+  std::stringstream given, toNormalize;
+  std::string noeLeadingSlash, twoChars1, twoChars2,slashOnly, uncOnly;
+
+
+  given << separator << separator << "SERVER" << separator << "fileName";
+  toNormalize << separator << "pathel" << separator << "pathel_1";
+
+  std::string expected = given.str();
+
+  ASSERT_EQ(expected, NormalizePath(given.str()));
```

Comment:
The tests might be clearer if you had ASSERT_EQ(XXX.str(), NormalizePath(XXX.str()))
where XXX is the string you are interested in.  

It might also be clearer if you just used string append rather than std::stream.  That way you can see the values
e.g (ASSERT_EQ(sep + sep + "SERVER" + sep + "fileName", NormailzePath(sep + sep  ...) ))